### ISSUE TITLE
Update dev files to make Vagrant and Docker startup work

### ISSUE THREAD

**Issue by lauralorenz** [DistrictDataLabs/cultivar#90]
Vagrant and docker dev environments don't start up properly:

- Vagrantfile is using base box's python 2 instead of python 3, which this codebase is (currently) python3 compatible only, starting with needing `urllib.parse`

```
cultivar $ vagrant up
# ....
==> default: /home/vagrant/project/venv/local/lib/python2.7/site-packages/six.py:808: RemovedInDjango110Warning: SubfieldBase has been deprecated. Use Field.from_db_value instead.
==> default:   return meta(name, bases, d)
==> default: Traceback (most recent call last):
==> default:   File "manage.py", line 39, in <module>
==> default:     
==> default: execute_from_command_line(sys.argv)
==> default:   File "/home/vagrant/project/venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
==> default:     
==> default: utility.execute()
==> default:   File "/home/vagrant/project/venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 327, in execute
==> default:     
==> default: django.setup()
==> default:   File "/home/vagrant/project/venv/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
==> default:     
==> default: apps.populate(settings.INSTALLED_APPS)
==> default:   File "/home/vagrant/project/venv/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
==> default:     
==> default: app_config.import_models(all_models)
==> default:   File "/home/vagrant/project/venv/local/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
==> default:     
==> default: self.models_module = import_module(models_module_name)
==> default:   File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
==> default:     
==> default: __import__(name)
==> default:   File "/home/vagrant/project/members/models.py", line 20, in <module>
==> default:     
==> default: import urllib.parse
==> default: ImportError
==> default: : 
==> default: No module named parse
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
cultivar $ 
```
- Docker compose file has container links to rabbitmq service but no rabbitmq service is defined. Possibly the dangling redis service was intended to replace it but if so the docker-compose as well as the rest of the default celery config needs to be updated accordingly, so it needs to go one way or the other:

```
cultivar $ docker-compose up -d
ERROR: Service 'celery' depends on service 'rabbitmq' which is undefined.
```

**Comment by lauralorenz** [comment]
Ah semi-jk on the docker one, I didn't read the directions and the one for dev is supposed to be `dev.yml`. Not sure if the `docker-compose.yml` should be booted or should have its rabbitmq service situation fixed, depending on what it's going to be used for.

**Comment by lauralorenz** [comment]
Closed by #91
